addons/pos_restaurant/static/tests/tours/pos_restaurant.js

```diff
@@ -30,6 +31,37 @@ function isSyncStatusConnected() {
         },
     ];
 }
+function checkLastOrderPreparationChange(expected_changes) {
+    return [
+        {
+            content: `Check last order preparation changes with expected changes ${JSON.stringify(expected_changes)}`,
+            trigger: ".pos", // dummy trigger
+            run: function() {
+                const currentOrder = posmodel.get_order();
+                const lastOrderPrepaChange = currentOrder.lastOrderPrepaChange;
+                const lastOrderPrepaChangesObj = Object.values(lastOrderPrepaChange);
+                
+                // Quick check for lenght
+                if (expected_changes.length !== lastOrderPrepaChangesObj.length) {
+                    console.error(`Was expecting ${expected_changes.length} order changes, got ${lastOrderPrepasChangeObj.length}`);
+                    debugger;
```

Comment:
you probably want to remove this `debugger`

Reply:
btw, this might not have been relevant for you here, but i find that the new features for testing introduced here are super nice https://github.com/odoo/odoo/pull/155082